### ISSUE TITLE
AFL: several builds started failing 15th December

### ISSUE THREAD

**Issue by DavidKorczynski** [google/oss-fuzz#7021]
dng_sdk started failing with AFL the 15th December and it looks to be an issue in the compiler:

```
Step #3 - "compile-afl-address-x86_64": 0.	Program arguments: /usr/local/bin/clang++ -Wno-unused-command-line-argument -Xclang -load -Xclang /src/aflplusplus/split-switches-pass.so -Xclang -load -Xclang /src/aflplusplus/compare-transform-pass.so -Xclang -load -Xclang /src/aflplusplus/split-compares-pass.so -fno-experimental-new-pass-manager -Xclang -load -Xclang /src/aflplusplus/afl-llvm-pass.so -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address -fsanitize-address-use-after-scope -stdlib=libc++ -DqDNGUseLibJPEG=1 -DqDNGUseXMP=0 -DqDNGThreadSafe=1 -c ./dng_1d_table.cpp -g -funroll-loops -fno-builtin-strcmp -fno-builtin-strncmp -fno-builtin-strcasecmp -fno-builtin-strncasecmp -fno-builtin-memcmp -fno-builtin-bcmp -fno-builtin-strstr -fno-builtin-strcasestr -D__AFL_HAVE_MANUAL_CONTROL=1 -D__AFL_COMPILER=1 -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION=1 "-D__AFL_FUZZ_INIT()=int __afl_sharedmem_fuzzing = 1;extern unsigned int *__afl_fuzz_len;extern unsigned char *__afl_fuzz_ptr;unsigned char __afl_fuzz_alt[1048576];unsigned char *__afl_fuzz_alt_ptr = __afl_fuzz_alt;" "-D__AFL_COVERAGE()=int __afl_selective_coverage = 1;extern \"C\" void __afl_coverage_discard();extern \"C\" void __afl_coverage_skip();extern \"C\" void __afl_coverage_on();extern \"C\" void __afl_coverage_off();" "-D__AFL_COVERAGE_START_OFF()=int __afl_selective_coverage_start_off = 1;" -D__AFL_COVERAGE_ON()=__afl_coverage_on() -D__AFL_COVERAGE_OFF()=__afl_coverage_off() -D__AFL_COVERAGE_DISCARD()=__afl_coverage_discard() -D__AFL_COVERAGE_SKIP()=__afl_coverage_skip() "-D__AFL_FUZZ_TESTCASE_BUF=(__afl_fuzz_ptr ? __afl_fuzz_ptr : __afl_fuzz_alt_ptr)" "-D__AFL_FUZZ_TESTCASE_LEN=(__afl_fuzz_ptr ? *__afl_fuzz_len : (*__afl_fuzz_len = read(0, __afl_fuzz_alt_ptr, 1048576)) == 0xffffffff ? 0 : *__afl_fuzz_len)" "-D__AFL_LOOP(_A)=({ static volatile char *_B __attribute__((used));  _B = (char*)\"##SIG_AFL_PERSISTENT##\"; __attribute__((visibility(\"default\"))) int _L(unsigned int) __asm__(\"__afl_persistent_loop\"); _L(_A); })" "-D__AFL_INIT()=do { static volatile char *_A __attribute__((used));  _A = (char*)\"##SIG_AFL_DEFER_FORKSRV##\"; __attribute__((visibility(\"default\"))) void _I(void) __asm__(\"__afl_manual_init\"); _I(); } while (0)" -Wno-unused-command-line-argument
Step #3 - "compile-afl-address-x86_64": 1.	<eof> parser at end of file
Step #3 - "compile-afl-address-x86_64": 2.	Per-module optimization passes
Step #3 - "compile-afl-address-x86_64": 3.	Running pass 'Unnamed pass: implement Pass::getPassName()' on module './dng_1d_table.cpp'.
Step #3 - "compile-afl-address-x86_64":  #0 0x000000000189b0b3 llvm::sys::PrintStackTrace(llvm::raw_ostream&, int) (/usr/local/bin/clang+++0x189b0b3)
Step #3 - "compile-afl-address-x86_64":  #1 0x00000000018990be llvm::sys::RunSignalHandlers() (/usr/local/bin/clang+++0x18990be)
Step #3 - "compile-afl-address-x86_64":  #2 0x000000000189a56d llvm::sys::CleanupOnSignal(unsigned long) (/usr/local/bin/clang+++0x189a56d)
Step #3 - "compile-afl-address-x86_64":  #3 0x000000000181f690 CrashRecoverySignalHandler(int) CrashRecoveryContext.cpp:0:0
Step #3 - "compile-afl-address-x86_64":  #4 0x00007f7fb2daa3c0 __restore_rt (/lib/x86_64-linux-gnu/libpthread.so.0+0x153c0)
Step #3 - "compile-afl-address-x86_64":  #5 0x00000000012c95fe bool llvm::DenseMapBase<llvm::DenseMap<llvm::FunctionType*, llvm::detail::DenseSetEmpty, llvm::FunctionTypeKeyInfo, llvm::detail::DenseSetPair<llvm::FunctionType*> >, llvm::FunctionType*, llvm::detail::DenseSetEmpty, llvm::FunctionTypeKeyInfo, llvm::detail::DenseSetPair<llvm::FunctionType*> >::LookupBucketFor<llvm::FunctionTypeKeyInfo::KeyTy>(llvm::FunctionTypeKeyInfo::KeyTy const&, llvm::detail::DenseSetPair<llvm::FunctionType*> const*&) const (/usr/local/bin/clang+++0x12c95fe)
Step #3 - "compile-afl-address-x86_64":  #6 0x00000000012c7cb4 llvm::FunctionType::get(llvm::Type*, llvm::ArrayRef<llvm::Type*>, bool) (/usr/local/bin/clang+++0x12c7cb4)
Step #3 - "compile-afl-address-x86_64":  #7 0x000000000124fff9 llvm::Intrinsic::getType(llvm::LLVMContext&, unsigned int, llvm::ArrayRef<llvm::Type*>) (/usr/local/bin/clang+++0x124fff9)
Step #3 - "compile-afl-address-x86_64":  #8 0x0000000001250592 llvm::Intrinsic::getDeclaration(llvm::Module*, unsigned int, llvm::ArrayRef<llvm::Type*>) (/usr/local/bin/clang+++0x1250592)
Step #3 - "compile-afl-address-x86_64":  #9 0x00000000012580c5 llvm::IRBuilderBase::CreateXorReduce(llvm::Value*) (/usr/local/bin/clang+++0x12580c5)
Step #3 - "compile-afl-address-x86_64": #10 0x00007f7fb27adaa5 (anonymous namespace)::AFLCoverage::runOnModule(llvm::Module&) /src/aflplusplus/instrumentation/afl-llvm-pass.so.cc:465:34
Step #3 - "compile-afl-address-x86_64": #11 0x0000000001289064 llvm::legacy::PassManagerImpl::run(llvm::Module&) (/usr/local/bin/clang+++0x1289064)
Step #3 - "compile-afl-address-x86_64": #12 0x0000000001ac6000 clang::EmitBackendOutput(clang::DiagnosticsEngine&, clang::HeaderSearchOptions const&, clang::CodeGenOptions const&, clang::TargetOptions const&, clang::LangOptions const&, llvm::StringRef, llvm::Module*, clang::BackendAction, std::unique_ptr<llvm::raw_pwrite_stream, std::default_delete<llvm::raw_pwrite_stream> >) (/usr/local/bin/clang+++0x1ac6000)
Step #3 - "compile-afl-address-x86_64": #13 0x0000000002606ee1 clang::BackendConsumer::HandleTranslationUnit(clang::ASTContext&) (/usr/local/bin/clang+++0x2606ee1)
Step #3 - "compile-afl-address-x86_64": #14 0x0000000002ff7014 clang::ParseAST(clang::Sema&, bool, bool) (/usr/local/bin/clang+++0x2ff7014)
Step #3 - "compile-afl-address-x86_64": #15 0x000000000207f7c9 clang::FrontendAction::Execute() (/usr/local/bin/clang+++0x207f7c9)
Step #3 - "compile-afl-address-x86_64": #16 0x0000000001ffcde6 clang::CompilerInstance::ExecuteAction(clang::FrontendAction&) (/usr/local/bin/clang+++0x1ffcde6)
Step #3 - "compile-afl-address-x86_64": #17 0x000000000213497b clang::ExecuteCompilerInvocation(clang::CompilerInstance*) (/usr/local/bin/clang+++0x213497b)
Step #3 - "compile-afl-address-x86_64": #18 0x000000000092ad74 cc1_main(llvm::ArrayRef<char const*>, char const*, void*) (/usr/local/bin/clang+++0x92ad74)
Step #3 - "compile-afl-address-x86_64": #19 0x0000000000928e68 ExecuteCC1Tool(llvm::SmallVectorImpl<char const*>&) driver.cpp:0:0
Step #3 - "compile-afl-address-x86_64": #20 0x0000000001ea7312 void llvm::function_ref<void ()>::callback_fn<clang::driver::CC1Command::Execute(llvm::ArrayRef<llvm::Optional<llvm::StringRef> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, bool*) const::$_1>(long) Job.cpp:0:0
Step #3 - "compile-afl-address-x86_64": #21 0x000000000181f43d llvm::CrashRecoveryContext::RunSafely(llvm::function_ref<void ()>) (/usr/local/bin/clang+++0x181f43d)
Step #3 - "compile-afl-address-x86_64": #22 0x0000000001ea6e6a clang::driver::CC1Command::Execute(llvm::ArrayRef<llvm::Optional<llvm::StringRef> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, bool*) const (/usr/local/bin/clang+++0x1ea6e6a)
Step #3 - "compile-afl-address-x86_64": #23 0x0000000001e773fd clang::driver::Compilation::ExecuteCommand(clang::driver::Command const&, clang::driver::Command const*&) const (/usr/local/bin/clang+++0x1e773fd)
Step #3 - "compile-afl-address-x86_64": #24 0x0000000001e77647 clang::driver::Compilation::ExecuteJobs(clang::driver::JobList const&, llvm::SmallVectorImpl<std::pair<int, clang::driver::Command const*> >&) const (/usr/local/bin/clang+++0x1e77647)
Step #3 - "compile-afl-address-x86_64": #25 0x0000000001e8e091 clang::driver::Driver::ExecuteCompilation(clang::driver::Compilation&, llvm::SmallVectorImpl<std::pair<int, clang::driver::Command const*> >&) (/usr/local/bin/clang+++0x1e8e091)
Step #3 - "compile-afl-address-x86_64": #26 0x000000000092857b main (/usr/local/bin/clang+++0x92857b)
Step #3 - "compile-afl-address-x86_64": #27 0x00007f7fb284f0b3 __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x270b3)
Step #3 - "compile-afl-address-x86_64": #28 0x0000000000925c1e _start (/usr/local/bin/clang+++0x925c1e)
Step #3 - "compile-afl-address-x86_64": clang-14: error: clang frontend command failed with exit code 139 (use -v to see invocation)
Step #3 - "compile-afl-address-x86_64": clang version 14.0.0 (https://github.com/llvm/llvm-project.git aee49255074fd4ef38d97e6e70cbfbf2f9fd0fa7)
Step #3 - "compile-afl-address-x86_64": Target: x86_64-unknown-linux-gnu
Step #3 - "compile-afl-address-x86_64": Thread model: posix
Step #3 - "compile-afl-address-x86_64": InstalledDir: /usr/local/bin
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: 
Step #3 - "compile-afl-address-x86_64": ********************
Step #3 - "compile-afl-address-x86_64": 
Step #3 - "compile-afl-address-x86_64": PLEASE ATTACH THE FOLLOWING FILES TO THE BUG REPORT:
Step #3 - "compile-afl-address-x86_64": Preprocessed source(s) and associated run script(s) are located at:
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: /tmp/dng_1d_table-22d678.cpp
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: /tmp/dng_1d_table-22d678.sh
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: 
Step #3 - "compile-afl-address-x86_64": 
Step #3 - "compile-afl-address-x86_64": ********************
Step #3 - "compile-afl-address-x86_64": PLEASE submit a bug report to https://bugs.llvm.org/ and include the crash backtrace, preprocessed source, and associated run script.
```

Link to build log: https://oss-fuzz-build-logs.storage.googleapis.com/log-d8b96b13-2c85-4bf6-92fe-33379b78ad46.txt

**Comment by DavidKorczynski** [comment]
CC @vanhauser-thc

**Comment by DavidKorczynski** [comment]
cjson looks to fail with the same issue. Log from 15th December: https://oss-fuzz-build-logs.storage.googleapis.com/log-93b41f0f-bf58-4dd6-9c25-444b5d7b4beb.txt

**Comment by jonathanmetzman** [comment]
Probably due to https://github.com/google/oss-fuzz/pull/7008

**Comment by jonathanmetzman** [comment]
CC @vanhauser-thc

**Comment by vanhauser-thc** [comment]
was llvm updated to the newest version? or is it still tagged to some weeks ago?
we had several larger changes for llvm 14 and I am not surprised if not everything is working.

**Comment by DavidKorczynski** [comment]
Still tagged to some weeks ago

**Comment by DavidKorczynski** [comment]
The projects and the issue I listed here was fixed - the builds all switched green today.

There might be some other AFL build issues, Qt in particular https://oss-fuzz-build-logs.storage.googleapis.com/log-dba215ac-0acd-4e2e-830b-f868b5d9de1c.txt which fails with a different log.

**Comment by vanhauser-thc** [comment]
> There might be some other AFL build issues, Qt in particular https://oss-fuzz-build-logs.storage.googleapis.com/log-dba215ac-0acd-4e2e-830b-f868b5d9de1c.txt which fails with a different log.

that is fixed too

**Comment by DavidKorczynski** [comment]
> > There might be some other AFL build issues, Qt in particular https://oss-fuzz-build-logs.storage.googleapis.com/log-dba215ac-0acd-4e2e-830b-f868b5d9de1c.txt which fails with a different log.
> 
> that is fixed too

Is that fix in https://github.com/google/oss-fuzz/pull/7026 ?

**Comment by vanhauser-thc** [comment]
> > > There might be some other AFL build issues, Qt in particular https://oss-fuzz-build-logs.storage.googleapis.com/log-dba215ac-0acd-4e2e-830b-f868b5d9de1c.txt which fails with a different log.
> > 
> > 
> > that is fixed too
> 
> Is that fix in #7026 ?

yes. that is written in the 2nd line of the PR text ;)

**Comment by DavidKorczynski** [comment]
Ah I didn't see the "qt" - cheers!

**Comment by rlohning** [comment]
Hi, Qt still/again failed today with a different stacktrace. Should I create a new issue about it? Or did the build still use an older image which didn't contain the fix?

Step #3 - "compile-afl-address-x86_64": 1.	<eof> parser at end of file
Step #3 - "compile-afl-address-x86_64": 2.	Code generation
Step #3 - "compile-afl-address-x86_64": 3.	Running pass 'Function Pass Manager' on module '/work/qt/src/gui/Gui_autogen/mocs_compilation.cpp'.
Step #3 - "compile-afl-address-x86_64": 4.	Running pass 'X86 DAG->DAG Instruction Selection' on function '@_Zeq9QVector2DS_'
Step #3 - "compile-afl-address-x86_64":  #0 0x0000000001879eb3 llvm::sys::PrintStackTrace(llvm::raw_ostream&, int) (/usr/local/bin/clang+++0x1879eb3)
Step #3 - "compile-afl-address-x86_64":  #1 0x0000000001877e0e llvm::sys::RunSignalHandlers() (/usr/local/bin/clang+++0x1877e0e)
Step #3 - "compile-afl-address-x86_64":  #2 0x000000000187936d llvm::sys::CleanupOnSignal(unsigned long) (/usr/local/bin/clang+++0x187936d)
Step #3 - "compile-afl-address-x86_64":  #3 0x00000000017fec9a (anonymous namespace)::CrashRecoveryContextImpl::HandleCrash(int, unsigned long) CrashRecoveryContext.cpp:0:0
Step #3 - "compile-afl-address-x86_64":  #4 0x00000000017fec3b (/usr/local/bin/clang+++0x17fec3b)
Step #3 - "compile-afl-address-x86_64":  #5 0x0000000001874927 llvm::sys::Process::Exit(int, bool) (/usr/local/bin/clang+++0x1874927)
Step #3 - "compile-afl-address-x86_64":  #6 0x0000000000934b92 (/usr/local/bin/clang+++0x934b92)
Step #3 - "compile-afl-address-x86_64":  #7 0x0000000001802751 llvm::report_fatal_error(llvm::Twine const&, bool) (/usr/local/bin/clang+++0x1802751)
Step #3 - "compile-afl-address-x86_64":  #8 0x0000000002504836 (/usr/local/bin/clang+++0x2504836)
Step #3 - "compile-afl-address-x86_64":  #9 0x0000000002503d8d (/usr/local/bin/clang+++0x2503d8d)
Step #3 - "compile-afl-address-x86_64": #10 0x00000000009d325d (anonymous namespace)::X86DAGToDAGISel::Select(llvm::SDNode*) X86ISelDAGToDAG.cpp:0:0
Step #3 - "compile-afl-address-x86_64": #11 0x00000000024fce1f llvm::SelectionDAGISel::DoInstructionSelection() (/usr/local/bin/clang+++0x24fce1f)
Step #3 - "compile-afl-address-x86_64": #12 0x00000000024fc4c0 llvm::SelectionDAGISel::CodeGenAndEmitDAG() (/usr/local/bin/clang+++0x24fc4c0)
Step #3 - "compile-afl-address-x86_64": #13 0x00000000024fb6e0 llvm::SelectionDAGISel::SelectAllBasicBlocks(llvm::Function const&) (/usr/local/bin/clang+++0x24fb6e0)
Step #3 - "compile-afl-address-x86_64": #14 0x00000000024f94e7 llvm::SelectionDAGISel::runOnMachineFunction(llvm::MachineFunction&) (/usr/local/bin/clang+++0x24f94e7)
Step #3 - "compile-afl-address-x86_64": #15 0x00000000009ca13d (anonymous namespace)::X86DAGToDAGISel::runOnMachineFunction(llvm::MachineFunction&) X86ISelDAGToDAG.cpp:0:0
Step #3 - "compile-afl-address-x86_64": #16 0x0000000000e8e35e llvm::MachineFunctionPass::runOnFunction(llvm::Function&) (/usr/local/bin/clang+++0xe8e35e)
Step #3 - "compile-afl-address-x86_64": #17 0x000000000127b9fd llvm::FPPassManager::runOnFunction(llvm::Function&) (/usr/local/bin/clang+++0x127b9fd)
Step #3 - "compile-afl-address-x86_64": #18 0x0000000001282d63 llvm::FPPassManager::runOnModule(llvm::Module&) (/usr/local/bin/clang+++0x1282d63)
Step #3 - "compile-afl-address-x86_64": #19 0x000000000127c59d llvm::legacy::PassManagerImpl::run(llvm::Module&) (/usr/local/bin/clang+++0x127c59d)
Step #3 - "compile-afl-address-x86_64": #20 0x0000000001aa0b84 clang::EmitBackendOutput(clang::DiagnosticsEngine&, clang::HeaderSearchOptions const&, clang::CodeGenOptions const&, clang::TargetOptions const&, clang::LangOptions const&, llvm::StringRef, llvm::Module*, clang::BackendAction, std::unique_ptr<llvm::raw_pwrite_stream, std::default_delete<llvm::raw_pwrite_stream> >) (/usr/local/bin/clang+++0x1aa0b84)
Step #3 - "compile-afl-address-x86_64": #21 0x00000000025dc891 clang::BackendConsumer::HandleTranslationUnit(clang::ASTContext&) (/usr/local/bin/clang+++0x25dc891)
Step #3 - "compile-afl-address-x86_64": #22 0x0000000002fb7284 clang::ParseAST(clang::Sema&, bool, bool) (/usr/local/bin/clang+++0x2fb7284)
Step #3 - "compile-afl-address-x86_64": #23 0x0000000002051209 clang::FrontendAction::Execute() (/usr/local/bin/clang+++0x2051209)
Step #3 - "compile-afl-address-x86_64": #24 0x0000000001fcfa76 clang::CompilerInstance::ExecuteAction(clang::FrontendAction&) (/usr/local/bin/clang+++0x1fcfa76)
Step #3 - "compile-afl-address-x86_64": #25 0x000000000210415b clang::ExecuteCompilerInvocation(clang::CompilerInstance*) (/usr/local/bin/clang+++0x210415b)
Step #3 - "compile-afl-address-x86_64": #26 0x0000000000934874 cc1_main(llvm::ArrayRef<char const*>, char const*, void*) (/usr/local/bin/clang+++0x934874)
Step #3 - "compile-afl-address-x86_64": #27 0x0000000000932d38 ExecuteCC1Tool(llvm::SmallVectorImpl<char const*>&) driver.cpp:0:0
Step #3 - "compile-afl-address-x86_64": #28 0x0000000001e7e722 void llvm::function_ref<void ()>::callback_fn<clang::driver::CC1Command::Execute(llvm::ArrayRef<llvm::Optional<llvm::StringRef> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, bool*) const::$_1>(long) Job.cpp:0:0
Step #3 - "compile-afl-address-x86_64": #29 0x00000000017fec1d llvm::CrashRecoveryContext::RunSafely(llvm::function_ref<void ()>) (/usr/local/bin/clang+++0x17fec1d)
Step #3 - "compile-afl-address-x86_64": #30 0x0000000001e7e27a clang::driver::CC1Command::Execute(llvm::ArrayRef<llvm::Optional<llvm::StringRef> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*, bool*) const (/usr/local/bin/clang+++0x1e7e27a)
Step #3 - "compile-afl-address-x86_64": #31 0x0000000001e4f40d clang::driver::Compilation::ExecuteCommand(clang::driver::Command const&, clang::driver::Command const*&) const (/usr/local/bin/clang+++0x1e4f40d)
Step #3 - "compile-afl-address-x86_64": #32 0x0000000001e4f657 clang::driver::Compilation::ExecuteJobs(clang::driver::JobList const&, llvm::SmallVectorImpl<std::pair<int, clang::driver::Command const*> >&) const (/usr/local/bin/clang+++0x1e4f657)
Step #3 - "compile-afl-address-x86_64": #33 0x0000000001e65758 clang::driver::Driver::ExecuteCompilation(clang::driver::Compilation&, llvm::SmallVectorImpl<std::pair<int, clang::driver::Command const*> >&) (/usr/local/bin/clang+++0x1e65758)
Step #3 - "compile-afl-address-x86_64": #34 0x0000000000932528 main (/usr/local/bin/clang+++0x932528)
Step #3 - "compile-afl-address-x86_64": #35 0x00007fcc421d70b3 __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x270b3)
Step #3 - "compile-afl-address-x86_64": #36 0x000000000092fbfe _start (/usr/local/bin/clang+++0x92fbfe)
Step #3 - "compile-afl-address-x86_64": clang-14: error: clang frontend command failed with exit code 70 (use -v to see invocation)
Step #3 - "compile-afl-address-x86_64": clang version 14.0.0 (https://github.com/llvm/llvm-project.git aee49255074fd4ef38d97e6e70cbfbf2f9fd0fa7)
Step #3 - "compile-afl-address-x86_64": Target: x86_64-unknown-linux-gnu
Step #3 - "compile-afl-address-x86_64": Thread model: posix
Step #3 - "compile-afl-address-x86_64": InstalledDir: /usr/local/bin
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: 
Step #3 - "compile-afl-address-x86_64": ********************
Step #3 - "compile-afl-address-x86_64": 
Step #3 - "compile-afl-address-x86_64": PLEASE ATTACH THE FOLLOWING FILES TO THE BUG REPORT:
Step #3 - "compile-afl-address-x86_64": Preprocessed source(s) and associated run script(s) are located at:
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: /tmp/mocs_compilation-0f4664.cpp
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: /tmp/mocs_compilation-0f4664.sh
Step #3 - "compile-afl-address-x86_64": clang-14: note: diagnostic msg: 
Step #3 - "compile-afl-address-x86_64": 
Step #3 - "compile-afl-address-x86_64": ********************

https://oss-fuzz-build-logs.storage.googleapis.com/log-e0fcacef-e7f7-4f6a-99f7-2e8c13e3cae4.txt

**Comment by rlohning** [comment]
Oh, I see the fix was not merged yet. So I'll wait and hope for that...

**Comment by rlohning** [comment]
You probably already noticed that afl is also being extremely slow. Even successful builds of Qt in oss-fuzz literally take hours to finish. Should I create a separate report for that or is there anything else I can do to help?

**Comment by vanhauser-thc** [comment]
well I need the fix merged first and then see if anything still pops up ... so if this still happens after #7026 then ping me.

**Comment by rlohning** [comment]
Now that the fix is in, building Qt for afl just takes about 10 percent longer than for libfuzzer. This is not a significant difference to me. Thank you for the fixes!

**Comment by DavidKorczynski** [comment]
All projects mentioned in this issue are now building, closing. Thanks for  https://github.com/google/oss-fuzz/pull/7026 @vanhauser-thc !
